.gitmodules

```diff
@@ -25,12 +22,18 @@
 [submodule "external/ghc-filesystem"]
 	path = external/ghc-filesystem
 	url = https://github.com/gulrak/filesystem.git
-[submodule "external/fmt"]
-	path = external/fmt
-	url = https://github.com/fmtlib/fmt.git
 [submodule "external/SQLiteCpp"]
 	path = external/SQLiteCpp
 	url = https://github.com/SRombauts/SQLiteCpp
 [submodule "external/oxen-encoding"]
 	path = external/oxen-encoding
 	url = https://github.com/oxen-io/oxen-encoding.git
+[submodule "external/oxen-logging"]
+	path = external/oxen-logging
+	url = https://github.com/oxen-io/oxen-logging
+[submodule "external/loki-mq"]
+	path = external/oxen-mq
+  url = https://github.com/oxen-io/loki-mq.git
+[submodule "oxen-logging"]
+	path = oxen-logging
+	url = https://github.com/oxen-io/oxen-logging
```

Comment:
This looks like a duplicate submodule; I assume it's the one in `external/` that should be here.